subworkflows/nf-scil/preproc_t1/main.nf

```diff
@@ -3,18 +3,22 @@ include { DENOISING_NLMEANS } from '../../../modules/nf-scil/denoising/nlmeans/m
 include { PREPROC_N4 } from '../../../modules/nf-scil/preproc/n4/main'
 include { IMAGE_RESAMPLE } from '../../../modules/nf-scil/image/resample/main'
 include { BETCROP_ANTSBET } from '../../../modules/nf-scil/betcrop/antsbet/main'
+include { BETCROP_SYNTHBET} from '../../../modules/nf-scil/betcrop/synthbet/main'
 include { BETCROP_CROPVOLUME as BETCROP_CROPVOLUME_T1 } from '../../../modules/nf-scil/betcrop/cropvolume/main'
 include { BETCROP_CROPVOLUME as BETCROP_CROPVOLUME_MASK } from '../../../modules/nf-scil/betcrop/cropvolume/main'
 
 workflow PREPROC_T1 {
 
     take:
         ch_image           // channel: [ val(meta), [ image ] ]
-        ch_template        // channel: [ val(meta), [ template ] ]
-        ch_probability_map // channel: [ val(meta), [ probability_map ] ]
+        ch_template        // channel: [ val(meta), [ template ] ]        , optional
+        ch_probability_map // channel: [ val(meta), [ probability_map ] ] , optional
         ch_mask_nlmeans    // channel: [ val(meta), [ mask ] ]            , optional
         ch_ref_n4          // channel: [ val(meta), [ ref, ref_mask ] ]   , optional
         ch_ref_resample    // channel: [ val(meta), [ ref ] ]             , optional
+        ch_weights         // channel: [ val(meta), [ weights ] ]         , optional
+
+        val_synth(false)   // value: (default: false)                     , optional
```

Comment:
Conditional execution should not be done using the wokrflow input. This should be removed and replaced by a parameter, aka `params.run_synth_bet`, located at the top of this file, outside the `workflow` declaration.